cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterCard/UnmanagedClusterCard.tsx

```diff
@@ -13,16 +13,22 @@ import { UnmanagedCluster } from '../../../swagger-api';
 
 ClarityIcons.addIcons(disconnectIcon, unknownStatusIcon);
 
+/* eslint-disable no-unused-vars */
 enum UnmanagedClusterStatus {
     RUNNING = 'RUNNING',
     STOPPED = 'STOPPED',
     UNKNOWN = 'UNKNOWN',
 }
 
-function UnmanagedClusterCard(props: UnmanagedCluster) {
-    const { name, provider, status } = props;
+interface UnmanagedClusterProps extends UnmanagedCluster {
+    confirmDeleteCallback: (arg?: string) => void;
```

Comment:
As mentioned over Zoom, I suggest having UnmanageClusterProps have two properties: cluster and confirmDeleteCallback, where cluster is of type UnmanagedCluster